components/Dialogs/src/MDCDialogPresentationController.h

```diff
@@ -42,6 +42,15 @@
  */
 @property(nonatomic, assign) BOOL dismissOnBackgroundTap;
 
+/**
+ Customize the corner radius in the tracking view to match the presented view's corner radius.
+ If the presented view and tracking view have different cornerRadius values, the rendered shadow
+ will not match.
+ 
+ Defaults to 0.0.
+ */
+@property(nonatomic, assign) CGFloat trackingViewCornerRadius;
```

Comment:
Can we simply call this `cornerRadius`? The fact that we are using a tracking view is an implementation detail that can easily change in the future.

Reply:
Naming is hard :/
s/trackingView/presentedView/
What do you think?

Reply:
Ah. Perhaps `dialogCornerRadius` then?